clients/src/test/java/org/apache/kafka/clients/consumer/internals/HeartbeatRequestManagerTest.java

```diff
@@ -425,6 +482,7 @@ topicId, mkSortedSet(0)
         assertEquals(Collections.singletonList(expectedTopicPartitions), heartbeatRequest1.data().topicPartitions());
 
         // Assignment did not change, so no assignment should be sent
```

Comment:
nit: I noticed that the `getHeartbeatRequest` used here has a kind of copy of this comment, totally out of place (ln 497). Could you pls remove it?